apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Tutorial/index.tsx

```diff
@@ -567,15 +571,29 @@ export default function Tutorial({ useCase }: Props) {
             deviceModelId: connectedDevice.modelId,
             fallbackIfNoAction: () => history.push("/"),
           });
-        if (upsellPath) {
+
+        if (useCase === UseCase.setupDevice && upsellPath) {
           history.push(upsellPath);
+        } else if (useCase === UseCase.recoveryPhrase && postOnboardingPath) {
+          history.push(postOnboardingPath);
+        } else if (useCase === UseCase.connectDevice && devicePairingPath) {
+          history.push(devicePairingPath);
```

Comment:
Was that added to move to different "route" if you end the onboarding in different screens ?